alerting/src/main/kotlin/org/opensearch/alerting/alerts/AlertIndices.kt

```diff
@@ -88,11 +103,20 @@ class AlertIndices(
         /** The index name pattern referring to all alert history indices */
         const val HISTORY_ALL = ".opendistro-alerting-alert-history*"
 
+        /** The index name pattern referring to all alert history indices */
+        const val FINDING_ALL = ".opendistro-alerting-alert-finding-history*"
+
         /** The index name pattern to create alert history indices */
         const val HISTORY_INDEX_PATTERN = "<.opendistro-alerting-alert-history-{now/d}-1>"
 
         /** The index name pattern to query all alerts, history and current alerts. */
-        const val ALL_INDEX_PATTERN = ".opendistro-alerting-alert*"
+        const val ALL_INDEX_PATTERN = ".opendistro-alerting-alert-alert*"
```

Comment:
Keep this the same and modify the pattern for findings as suggested. Also change the name to `ALL_ALERT_INDEX_PATTERN`. Make a copy of this for Findings with: `const val ALL_FINDING_INDEX_PATTERN = ".opendistro-alerting-finding*"`